website/i18n/en.json

```diff
@@ -0,0 +1,4325 @@
+{
```

Comment:
This file no longer exist in V2 (i18n will be added in the future), so please remove it form this changeset.

Reply:
done